mivs/model_checks.py

```diff
@@ -2,6 +2,9 @@
 
 
 def _is_invalid_url(url):
+    if not c.VALIDATE_SUBMITTED_URLS:
+        return False
```

Comment:
+1 on this review, but these three lines would be better if the setting was ``c.SKIP_MIVS_URL_VALIDATION``.  I had to read this several times, because these three lines make a triple negative:
- we're returning ``False``
- because the ``not`` check in the if statement
- in the ``invalid`` url function

This is an example of one of those things which seems obvious when you're writing it, and then someone else is like, "huh, a triple negative... lemme think REALLY hard about that".